### PR TITLE
Complete the code in example doc

### PR DESCRIPTION
**Pull request by lenjoy**
Complete the code in example doc, also make the code work for both python2 and python3

### CODE REVIEW

File: Doc/src/examples.rst [hunk 1]
```diff
@@ -87,19 +105,20 @@ As in the first example, we use the EAX mode to allow detection of unauthorized
...
+    enc_session_key = cipher_rsa.encrypt(session_key)
 
-    # Encrypt the data with the AES session key
+    # Encrypt the data with the AES session key                                                                                                                                                                  
     cipher_aes = AES.new(session_key, AES.MODE_EAX)
     ciphertext, tag = cipher_aes.encrypt_and_digest(data)
-    [ file_out.write(x) for x in (cipher_aes.nonce, tag, ciphertext) ]
+    [ file_out.write(x) for x in (enc_session_key, cipher_aes.nonce, tag, ciphertext) ]
 
 The receiver has the private RSA key. They will use it to decrypt the session key
```
Comment (frispete) [inline Doc/src/examples.rst]: Just by looking at this patch, doesnm't this comment contradicts the comment in line 75?

Comment (lenjoy) [reply Doc/src/examples.rst]: which two parts are contradicting to each other?

Comment (frispete) [reply Doc/src/examples.rst]: Line 75: The following code generates public key stored in ``receiver.pem`` and private key stored in ``private.pem``.
Line 123: The receiver has the private RSA key

Comment (Legrandin) [reply Doc/src/examples.rst]: Right. Probably two better filenames are `rx_public.pem` and `rx_private.pem`?

**Pull request merged**